fuzzers/aflplusplus_optimal/fuzzer.py

```diff
@@ -31,35 +31,37 @@ def build():  # pylint: disable=too-many-branches,too-many-statements
     """Build benchmark."""
     benchmark_name = os.environ['BENCHMARK']
 
-    if benchmark_name == 'bloaty_fuzz_target':
-        aflplusplus_fuzzer.build("lto", "laf", "autodict", "dynamic")
-    elif benchmark_name == 'curl_curl_fuzzer_http':
+    if benchmark_name == 'curl_curl_fuzzer_http':
```

Comment:
Do you want anything for bloaty, that benchmark is still there.

Reply:
bloaty is using the default now (LTO with a dynamic map and cmplog/redqueen)